### DISCUSSION TITLE
Reverse proxy configuration

### DISCUSSION THREAD

**Discussion by zyiqian** [Kong/kong#13772] [Help] [answered]
I am using kong Controller to deploy on k8s cluster, everything is working fine, but I am a little confused when configuring reverse proxy. I don't know how to do it through ingress or kongingress. 

My goal is that when I request the example-a.com domain, it will be proxied to the example-b.com domain

Can anyone help me? thanks

kong version: 3.1

**Comment by outsinre** [comment]
> it will be proxied to the example-b.com domain

The `example-b.com` should be configured in a service object. Please refer to https://docs.konghq.com/gateway/latest/get-started/services-and-routes/

**Reply by zyiqian** [reply]
hi @outsinre , Thank you very much for your quick reply. I am using the dbless method and don't know how to perform the operations in the document you provided.

**Reply by Oyami-Srk** [reply]
Hi @zyiqian, you can refer to [declarative config](https://docs.konghq.com/gateway/3.8.x/production/deployment-topologies/db-less-and-declarative-config/) to check how to configure gateway in DB-less mode. You may need to setup a service and route by host to achieve your goal.

**Reply by zyiqian** [reply]
Hi @Oyami-Srk , we are using the declarative config to configure, and are still verifying

**Comment by zyiqian** [comment, marked as answer]
I have already done it by injecting NGINX config.

```
server {
  listen 8081;
  server_name example-a.com;
  location / {
      proxy_pass http://example-b.com:80;
      proxy_set_header Connection "";
      proxy_set_header X-Real-Ip $remote_addr;
      proxy_set_header remoteip $remote_addr;
      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  }
}
```
